docs/source/developer/contributing.rst

```diff
@@ -80,51 +80,54 @@ case, please prepend ``[WIP]`` to your pull request's title.
 Build environment
 '''''''''''''''''
 
-Numba has a number of dependencies (mostly `Numpy <http://www.numpy.org/>`_
```

Comment:
Just noticed that 2-3 lines above this there's a `platorm` which would benefit from an 'f'.